beta/src/components/MDX/Sandpack/NavigationBar.tsx

```diff
@@ -41,6 +48,11 @@ export function NavigationBar({
     return;
   }, [openPaths.length, resizeHandler]);
 
+  const onReset = () => {
```

Comment:
Shift the approach to use the Sandpack API

Reply:
I recall trying `resetAllFiles` and it wasn't enough for some reason. Maybe I was missing `refresh` though.